src/tests/tablet/homeUI/CurrentMeetingValuesVerifications.java

```diff
@@ -0,0 +1,158 @@
+package tests.tablet.homeUI;
+
+import static tests.tablet.homeUI.HomeTabletSettings.createMeetingFromExcel;
+import static tests.tablet.homeUI.HomeTabletSettings.getAuthenticationValue;
+import static tests.tablet.homeUI.HomeTabletSettings.getMeetingSubject;
+import static tests.tablet.homeUI.HomeTabletSettings.getRoomDisplayName;
+
+import java.io.IOException;
+
+import org.apache.log4j.Logger;
+import org.apache.log4j.PropertyConfigurator;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+
+/**
+ * This TCs verify that {now} tile is set whit correct information
+ * when a meeting created in the current time.
+ * TC01: Meeting subject.
+ * TC02: Meeting organizer.
+ * TC31: Now tile is reed.
+ * TC30: Next tile is green.
+ * TC18: Available message is displayed in {Next} tile.
+ * TC07: Interval displayed in next tile is: end time current meeting - 0:00.
+ * @author Eliana Navia
+ * 
+ */
+public class CurrentMeetingValuesVerifications {
+	Logger log = Logger.getLogger(getClass());
+	HomeTabletPage homeTabletPage = new HomeTabletPage();
+	SchedulePage schedulePage;
+	String actualMeetingSubject = getMeetingSubject(0);
+	String actualMeetingOrganizer = "";
+	String expectedStartTime = "";
+
+	/**
+	 * 1. Meeting in progress in the room.
+	 * 2. Room free after meeting in progress.
+	 */
+	@BeforeClass
+	public void init() {
+		schedulePage = homeTabletPage.clickScheduleBtn();
+		createMeetingFromExcel(0);
+
+		schedulePage.clickOverMeetingCreated(actualMeetingSubject);
+		actualMeetingOrganizer = schedulePage.getMeetingOrganizerValue();
+		expectedStartTime = schedulePage.getEndTimeTxtBoxValue().substring(0, 5);
+		schedulePage.clickBackBtn();
+	}
+
+	/**
+	 * [EN] Verify that meeting subject is displayed on Now Tile 
+	 * when a meeting is in progress in the room.
+	 * TC01
+	 */
+	@Test (groups = {"ACCEPTANCE"})
```

Comment:
In this case the symbols "{}" are not necesary, because this test case only belong a one group 

correct that in all of your TC's